### DISCUSSION TITLE
Grouped Figures

### DISCUSSION THREAD

**Discussion by Laserist202277** [jgraph/drawio-desktop#1828] [Q&A] [answered]
Why when I drag new shapes to a grouped shape they are automatically added to the group? Is there any way to switch this off in version 20.3.0?

**Comment by alderg** [comment, marked as answer]
Try `dropTarget=0`

**Reply by Laserist202277** [reply]
Where should I use it?

**Reply by Laserist202277** [reply]
Thanks, found where it is. Can I use it for all groups on page? Or only always for each one?
